client/src/components/ProfileReview/ProfileReview.tsx

```diff
@@ -0,0 +1,25 @@
+import { ListItem, ListItemAvatar, ListItemText, Rating } from '@mui/material';
+import Review from '../../interface/Review';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+
+interface Props {
+  review: Review;
+}
+
+export default function ProfileReview({ review }: Props): JSX.Element {
+  return (
+    <>
+      <Rating value={review.rating} readOnly />
+      <ListItem sx={{ padding: '0', alignItems: 'flex-start' }}>
+        <ListItemAvatar>
+          <AvatarDisplay
+            loggedIn
+            user={{ name: review.reviewer.name, email: 'example@example.com' }}
```

Comment:
Why are we passing a hardcoded email as a prop here? :)

Reply:
AvatarDisplay is using the User interface which requires an email despite not using the email for anything. Instead of changing either and breaking a bunch of code I thought this would be alright. The root issue should probably be addressed instead though.